### DISCUSSION TITLE
classes/class_area3d

### DISCUSSION THREAD

**Discussion by giscus** [godotengine/godot-docs-user-notes#155] [User-contributed notes]
# classes/class_area3d

Inherits: CollisionObject3D< Node3D< Node< Object A region of 3D space that detects other CollisionObject3D s entering or exiting it. Description: Area3D is a region of 3D space defined by one or m...

https://docs.godotengine.org/en/stable/classes/class_area3d.html

**Comment by ThaddeusWilliams** [comment]
Warning: There is a bug in 4.3 "stable" that makes area3D nodes not collide properly if animated...

**Reply by Calinou** [reply]
This is a known issue:

- https://github.com/godotengine/godot/issues/74300

**Comment by MartinDelille** [comment]
I would like to use the wind properties to animate the sail of my boat but I have several question:
- how can I make it come from a direction an not a point ?
- why moving the source object position and direction doesn't affect the SoftBody3D ?

**Comment by v1993** [comment]
It would be really nice to have a more elaborate explanation of how to ensure that areas and bodies created directly via `PhysicsServer3D` work as expected (or in what cases they may break). Right now, the warning is so vague that it implies that you just shouldn't do that.
